packages/pwa-kit-react-sdk/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## v3.9.0-dev (Oct 29, 2024)
+- Add support for redirects with HTTP status other than 302 [#2173](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2173)
```

Comment:
It would be nice to call out the new method in here. Something like "Added `RedirectWithStatus` component allowing finer grained control of redirects and their status code" .. so if developers read this, they will know there is a new component.